test/etools-datepicker_test.html

```diff
@@ -20,10 +21,27 @@
 
     <script>
       suite('etools-datepicker', function() {
+        var basic
+        setup(function() {
+          basic = fixture('basic')
+        });
         test('instantiating the element works', function() {
-          var element = fixture('basic');
-          assert.equal(element.is, 'etools-datepicker');
+          assert.equal(basic.is, 'etools-datepicker');
+        });
+        test('tapping datepicker icon toggles it open', function() {
+          expect(dialog.attributes[4].name).to.equal('aria-hidden')
+          var calendarIcon = Polymer.dom(basic.root).querySelector('iron-icon')
+          MockInteractions.tap(calendarIcon)
+          expect(dialog.attributes[4].name).to.not.equal('aria-hidden')
         });
+        test('tapping a date selects it', function() {
+          var calendarIcon = Polymer.dom(basic.root).querySelector('iron-icon')
+          MockInteractions.tap(calendarIcon)
+          var firstDay = dialog.children.picker.querySelector('.day-item.selection')
+          var firstDayValue = firstDay.attributes.date.value
```

Comment:
try to use `let` instead of `var`

Reply:
happy to use let! i thought we couldn't use es6 in these components?

Reply:
well... `let` is supported by IE11 too, so we do not need to worry about it. IE11 is a stupid old bowser we have to support (not for long) :)